packages/Frontend/src/containers/AuthenticateContainer.jsx

```diff
@@ -23,47 +22,78 @@ class AuthenticateContainer extends Component {
     super(props);
     this.state = {
       email: '',
-      error: '',
+      error: false,
       loading: false,
       password: '',
     };
   }
 
+  componentDidMount = () => {
+    this.setState({ loading: true }, () => {
+      auth().getRedirectResult().then(() => {
+        this.setState({ loading: false });
+      }, (error) => {
+        this.setState({ loading: false });
+        if (error.code === 'auth/account-exists-with-different-credential') {
+          // eslint-disable-next-line no-unused-vars
+          auth().fetchProvidersForEmail(error.email).then((providers) => {
```

Comment:
We need to handle this later, not a big deal if we leave it undone for now. You could make a ticket for it though. Here's an example of what could happen. If someone signs up with Facebook and their Facebook account has the email, joe@somebody.com and then later tries to sign up with Google and their email is also joe@somebody.com with Google then we get an error, we can't have two account with the same email address. My comment below suggests what we should do to handle it.

Reply:
Added ticket; delete comment.